vk_video_decoder/libs/VkVideoDecoder/VkVideoDecoder.cpp

```diff
@@ -549,6 +537,11 @@ int VkVideoDecoder::DecodePictureWithParameters(VkParserPerFrameDecodeParameters
             assert(imageBarriers[numDpbBarriers].image);
             numDpbBarriers++;
         }
+        }
+    if ((m_capabilityFlags & VK_VIDEO_DECODE_CAPABILITY_DPB_AND_OUTPUT_DISTINCT_BIT_KHR)) {
+        pPicParams->pictureResources[pPicParams->numGopReferenceSlots] = pPicParams->decodeFrameInfo.dstPictureResource;
+        pPicParams->decodeFrameInfo.dstPictureResource = currentOutputPictureResource;
+
     }
```

Comment:
I don't think this function would need any modifications anymore, right?

Reply:
i think this change is required